client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -1,152 +1,202 @@
 
-		return <div className="plugins-browser-item__pricing">{ translate( 'Free' ) }</div>;
-	}
+	const isUntestedVersion = useMemo( () => {
+		const wpVersion = selectedSite?.options?.software_version;
+		const pluginTestedVersion = plugin?.tested;
 
-	renderUpgradeButton() {
-		const { isPreinstalled, upgradeLink } = this.props.plugin;
-		if ( isPreinstalled || ! upgradeLink ) {
-			return null;
+		if ( ! wpVersion || ! pluginTestedVersion ) {
+			return false;
 		}
 
-		return (
-			<Button className="plugins-browser-item__upgrade-button" compact primary href={ upgradeLink }>
-				{ this.props.translate( 'Upgrade' ) }
-			</Button>
-		);
-	}
+		return version_compare( wpVersion, pluginTestedVersion, '>' );
+	} );
 
-	renderPlaceholder() {
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
-		return (
-			<li className="plugins-browser-item is-placeholder">
-				<span className="plugins-browser-item__link">
-					<div className="plugins-browser-item__info">
-						<PluginIcon size={ this.props.iconSize } isPlaceholder={ true } />
-						<div className="plugins-browser-item__title">…</div>
-						<div className="plugins-browser-item__author">…</div>
-						<div className="plugins-browser-item__description">…</div>
-					</div>
-				</span>
-			</li>
-		);
-		/* eslint-enable wpcalypso/jsx-classname-namespace */
+	if ( isPlaceholder ) {
+		return <Placeholder iconSize={ iconSize } />;
 	}
 
-	render() {
-		const { isPlaceholder, plugin, iconSize, variant, translate } = this.props;
-		if ( isPlaceholder ) {
-			return this.renderPlaceholder();
-		}
-
-		const classNames = classnames( 'plugins-browser-item', variant );
-
-		return (
-			<li className={ classNames }>
-				<a
-					href={ this.getPluginLink() }
-					className="plugins-browser-item__link"
-					onClick={ this.trackPluginLinkClick }
-				>
-					<div className="plugins-browser-item__info">
-						<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
-						<div className="plugins-browser-item__title">{ plugin.name }</div>
-						{ variant === PluginsBrowserElementVariant.Extended && (
+	const classNames = classnames( 'plugins-browser-item', variant );
+	return (
+		<li className={ classNames }>
+			<a
+				href={ pluginLink }
+				className="plugins-browser-item__link"
+				onClick={ trackPluginLinkClick }
+			>
+				<div className="plugins-browser-item__info">
+					<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
+					<div className="plugins-browser-item__title">{ plugin.name }</div>
+					{ variant === PluginsBrowserElementVariant.Extended && (
+						<>
 							<div className="plugins-browser-item__author">
 								{ translate( 'by ' ) }
 								<span className="plugins-browser-item__author-name">{ plugin.author_name }</span>
 							</div>
+
+							<div className="plugins-browser-item__last-updated">
+								{ dateFromNow && (
+									<>
+										{ translate( 'Last updated ' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **144** times:
`translate( 'Last Updated:' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/enhance-plugins-browser-item)